zcash_primitives/src/sapling/spec.rs

```diff
@@ -98,6 +98,19 @@ pub(crate) fn ka_sapling_derive_public_prepared(
     b * sk
 }
 
+/// This is defined implicitly by [Zcash Protocol Spec § 4.2.2: Sapling Key Components][saplingkeycomponents]
+/// which uses $KA^\mathsf{Sapling}.\mathsf{DerivePublic}$ to produce a diversified
+/// transmission key with type $KA^\mathsf{Sapling}.\mathsf{PublicPrimeSubgroup}$.
```

Comment:
This will also change back to $\mathsf{KA^{Sapling}.PublicPrimeOrder}$. (Not needed in this PR, just making a note for myself.) https://github.com/zcash/zips/issues/664

Reply:
Deferred to #762.